### PR TITLE
Feature/7 registration component

### PR DESCRIPTION
**Pull request by cmendezlopez25**

### CODE REVIEW

File: RideShare/src/app/registration/registration.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    expect(compiled.querySelector('#email').name).toContain('userEmail');
+  });
+
+  it('should contain password input', () => {
+    const compiled = fixture.debugElement.nativeElement;
+    expect(compiled.querySelector('#password').name).toContain('userPassword');
+  });
+
+  /*
+  it('should call onBack method', () => {
```
Comment (jurczakn) [inline RideShare/src/app/registration/registration.component.spec.ts]: remove commented out test

**Review by jurczakn — CHANGES_REQUESTED** [review]

File: RideShare/src/app/registration/registration.component.spec.ts (hunk 1 above)
Comment (Monjoie11) [reply RideShare/src/app/registration/registration.component.spec.ts]: text has been commented out with requested change pushed

Comment (Monjoie11) [reply RideShare/src/app/registration/registration.component.spec.ts]: I mean commented test has been deleted

**Review by jurczakn — APPROVED** [review]
perfect

**Pull request merged**